sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/Block.java

```diff
@@ -25,7 +25,13 @@
      * The block size in bytes.
      */
     @JsonProperty(value = "Size", required = true)
-    private int size;
+    private long sizeLong;
+
+    /*
+     * The size property.
+     */
```

Comment:
That's not harmful remnant of swagger generation. See `swagger/readme.md` change in this PR. In order to transition from int->long I had to generate two Java properties for same field in such a way that one is actually used by Jackson parser to deserialize incoming xml and other to assure backwards compatibility - then remove data binding annotations and body so that extra property doesn't take part in deserialization process.

This comment is remnant of this process. I didn't manage to invent right regex that could get rid of this yet. But I can try harder.

Reply:
removed..